panels/history/ha-panel-history.html

```diff
@@ -61,22 +62,22 @@
           <vaadin-date-picker
             id='picker'
             value='{{_currentDate}}'
-            label='Showing entries for'
+            label="[[haLocalize('ui.panel.history', 'showing_entries')]]"
             disabled='[[isLoadingData]]'
           ></vaadin-date-picker>
 
           <paper-dropdown-menu
             label-float
-            label='Period'
+            label="[[haLocalize('ui.panel.history', 'period')]]"
             disabled='[[isLoadingData]]'
           >
             <paper-listbox
               slot="dropdown-content"
               selected="{{_periodIndex}}"
             >
-              <paper-item>1 day</paper-item>
-              <paper-item>3 days</paper-item>
-              <paper-item>1 week</paper-item>
+              <paper-item>[[haLocalize('ui.panel.history.periods', 'day', 'count', 1)]]</paper-item>
```

Comment:
Shouldn't this be `haLocalize('ui.panel.history.periods.day', 'count', 1)` ?

Reply:
I agree.

I'm going to open a separate PR and just refactor away the `haLocalize` wrapper. I think I had catered it mostly to the original translation use-case, and most of the method has already been eliminated. At this point it's just more confusing/complicated than using the Polymer `localize` directly.